### PR TITLE
Doc updates for v1.0.1 release

### PR DESCRIPTION
**Pull request by wdurairaj**
- Made `cpg`, `provisioning_type` to optional
- Added SPOCK updates
- Added `importVol` option in StorageClass parameters.

### CODE REVIEW

**Comment by datamattsson** [comment]
Just a FYI, @c-snell is on PTO this week. Don't expect a review until next week.

**Comment by datamattsson** [comment]
Just a FYI, @c-snell is on PTO this week. Don't expect a review until next week.

File: docs/container_storage_provider/hpe_3par_primera/index.md [hunk 1]
```diff
@@ -77,13 +77,14 @@ These parameters are used for volume provisioning and supported platforms.
 | --------------------------------- | ------- | ----------- | :--: | :-----: |
 | accessProtocol <br /> (required)    | fc      | The access protocol to use when accessing the persistent volume. | **X** | **X** |
 |                                     | iscsi   | The access protocol to use when accessing the persistent volume. | **X** |   |
-| cpg <br /> (required)               | Text    | The name of existing CPG to be used for volume provisioning. | **X** | **X** | 
+| cpg <br />                | Text    | The name of existing CPG to be used for volume provisioning. If this parameter is not specified, CSP autocomputes this value| **X** | **X** |
```
Comment (c-snell) [inline docs/container_storage_provider/hpe_3par_primera/index.md:80]: grammar tweak: If the **cpg** parameter is not specified, the CSP will automatically set cpg parameter based upon a CPG available to 3PAR or Primera array.

File: docs/container_storage_provider/hpe_3par_primera/index.md [hunk 2]
```diff
@@ -77,13 +77,14 @@ These parameters are used for volume provisioning and supported platforms.
...
 | accessProtocol <br /> (required)    | fc      | The access protocol to use when accessing the persistent volume. | **X** | **X** |
 |                                     | iscsi   | The access protocol to use when accessing the persistent volume. | **X** |   |
-| cpg <br /> (required)               | Text    | The name of existing CPG to be used for volume provisioning. | **X** | **X** | 
+| cpg <br />                | Text    | The name of existing CPG to be used for volume provisioning. If this parameter is not specified, CSP autocomputes this value| **X** | **X** | 
 | snap_cpg                            | Text    | The name of the snapshot CPG to be used for volume provisioning. Defaults to value of `cpg` if not specified. | **X** | **X** |
 | compression                         | Boolean | Indicates that the volume should be compressed. | **X** |   |
-| provisioning_type <br /> (required) | tpvv    | Indicates Thin provisioned volume type. | **X** | **X** |
+| provisioning_type <br />  | tpvv    | Indicates Thin provisioned volume type. If this parameter is not specified, defaults to tpvv | **X** | **X** |
```
Comment (c-snell) [inline docs/container_storage_provider/hpe_3par_primera/index.md:83]: Default: tpvv

**Comment by wdurairaj** [comment]
@c-snell, i have addressed review comments. Please review

**Pull request merged**